inc/custom-taxonomies.php

```diff
@@ -232,4 +232,121 @@ function feedimporter_job_region_custom_taxonomy() {
+
+	$labels = array(
+		'name'                       => 'Prison Types',
+		'singular_name'              => 'Prison Type',
+		'menu_name'                  => 'Prison Types',
+		'all_items'                  => 'All Prison Types',
+		'parent_item'                => 'Parent Prison Type',
+		'parent_item_colon'          => 'Parent Prison Type:',
+		'new_item_name'              => 'New Prison Type Name',
+		'add_new_item'               => 'Add New Prison Type',
+		'edit_item'                  => 'Edit Prison Type',
+		'update_item'                => 'Update Prison Type',
+		'view_item'                  => 'View Prison Type',
+		'separate_items_with_commas' => 'Separate prisons types with commas',
+		'add_or_remove_items'        => 'Add or remove prison types',
+		'choose_from_most_used'      => 'Choose from the most used',
+		'popular_items'              => 'Popular Prison Types',
+		'search_items'               => 'Search Prison Types',
+		'not_found'                  => 'Not Found',
+		'no_terms'                   => 'No Prison Types',
+		'items_list'                 => 'Prison Types list',
+		'items_list_navigation'      => 'Prison Types list navigation',
+	);
+	$args = array(
+		'labels'                     => $labels,
+		'hierarchical'               => true,
+		'public'                     => true,
+		'show_ui'                    => true,
+		'show_admin_column'          => true,
+		'show_in_nav_menus'          => true,
+		'show_tagcloud'              => true,
+	);
+	register_taxonomy( 'job_prison_type', array( 'job' ), $args );
+
+}
+add_action( 'init', 'feedimporter_prison_type_custom_taxonomy', 0 );
+
+// Register Prison Category Custom Taxonomy
+function feedimporter_prison_category_custom_taxonomy() {
+
+	$labels = array(
+		'name'                       => 'Prison Categories',
+		'singular_name'              => 'Prison Category',
+		'menu_name'                  => 'Prison Categories',
+		'all_items'                  => 'All Prison Categories',
+		'parent_item'                => 'Parent Prison Category',
+		'parent_item_colon'          => 'Parent Prison Category:',
+		'new_item_name'              => 'New Prison Category Name',
+		'add_new_item'               => 'Add New Prison Category',
+		'edit_item'                  => 'Edit Prison Category',
+		'update_item'                => 'Update Prison Category',
+		'view_item'                  => 'View Prison Category',
+		'separate_items_with_commas' => 'Separate prisons categories with commas',
+		'add_or_remove_items'        => 'Add or remove prison categories',
+		'choose_from_most_used'      => 'Choose from the most used',
+		'popular_items'              => 'Popular Prison Categories',
+		'search_items'               => 'Search Prison Categories',
+		'not_found'                  => 'Not Found',
+		'no_terms'                   => 'No Prison Categories',
+		'items_list'                 => 'Prison Categories list',
+		'items_list_navigation'      => 'Prison Categories list navigation',
```

Comment:
Inconsistent capitalisation of "Prison Category".  (suggest lower case is used except for first letter)

> Add new prison category
Prison categories list

I can see it was inconsistent on previous taxonomies, maybe that should be fixed too.